shared/js/ui/models/site.es6.js

```diff
@@ -191,6 +197,15 @@ Site.prototype = window.$.extend({},
       return count
     },
 
+    getMajorTrackerNetworksCount: function () {
+      // console.log('[model] getMajorTrackersCount()')
+      return Object.keys(this.tab.trackers).reduce((total, name) => {
+        let tempTracker = name.toLowerCase()
+        total += window.constants.majorTrackingNetworks[tempTracker] ? 1 : 0
+        return total
+      }, 0)
+    },
```

Comment:
Nice!!!

Reply:
The one thing I'd add here: As a practitioner of ultra-defensive coding, we should not assume that `constants.majorTrackingNetworks` will always be lower case. So can we add a `.toLowerCase()` for good measure to `window.constants.majorTrackingNetworks[tempTracker]` in case someone comes along later and is a bit sloppy/doesn't realize when they add a new tracking network uppercase?